nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -67,6 +67,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   `globalRedirect` can now have redirect codes other than 301 through
   `redirectCode`.
 
+- [](#opt-boot.kernel.sysctl) is defined as a freeformType and adds a custom merge option for [`"net.core.wmem_max"`](#opt-boot.kernel.sysctl._net.core.wmem_max_) (taking the highest value defined to avoid conflicts between 2 services trying to set that value), just as [`"net.core.rmem_max"`](#opt-boot.kernel.sysctl._net.core.rmem_max_) since 22.11.
```

Comment:
The fact that boot.kernel.sysctl is defined as a freeformType is not a change of this PR so I do not think this should be mentioned.

In addition, it would be good to mention the change of the option type because it is also a breaking change.

Reply:
I reworded this